diagnosisApplication/src/main/view/DiagnosisView.java

```diff
@@ -35,15 +35,16 @@ public DiagnosisView(DiagnosisViewModel diagnosisViewModel, DiagnosisController
             JPanel barChart = new JPanel();
             int[] accuracies = new int[n];
             String[] diagnoses = new String[n];
-            accuracies[0] = diagnosisState.getDiagnosis1Accuracy();
-            diagnoses[0] = diagnosisState.getDiagnosis1Name();
+
+            accuracies[0] = (Integer) diagnosisState.getDiagnosis1("accuracy");
+            diagnoses[0] = (String) diagnosisState.getDiagnosis1("name");
```

Comment:
Setting up values stored in hash maps for use through getters to be displayed in the view.